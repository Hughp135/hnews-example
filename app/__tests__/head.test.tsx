import { render } from "@testing-library/react";
import Head from "../head";

describe("Page", () => {
  it("renders the head", () => {
    const { asFragment } = render(<Head />);

    expect(asFragment()).toMatchSnapshot();
  });
});
