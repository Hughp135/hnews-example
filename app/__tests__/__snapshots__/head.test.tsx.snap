// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders the head 1`] = `
<DocumentFragment>
  <title>
    Create Next App
  </title>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    content="Generated by create next app"
    name="description"
  />
  <link
    href="/favicon.ico"
    rel="icon"
  />
</DocumentFragment>
`;
